pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
Also, should I also check the above "AllAlpha" gate ? It seems to imply all Alpha features enabled.

How about prefixing with Allow/Block ?

For 1.4, "AllowExtTrafficLocalEndpointAnnotation" ?

And in the future, negative behaviour, "BlockExtTrafficLocalEndpointAnnotation" ?